research.html

```diff
@@ -39,7 +39,10 @@
   <link rel="stylesheet" type="text/css" href="assets/css/research.css" />
   <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
   <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+  <script src="https://unpkg.com/splitting/dist/splitting.js"></script>
```

Comment:
In all the html files,

the scripts/stylesheet calls should be at the same position and with same comments.